jim-github.js

```diff
@@ -153,6 +153,7 @@ function createIssue(github, username, token, repository, issue, comments, miles
         if (!(issue.assignee in collaborators)) {
             // Add a comment storing the old assignee who is not a collaborator
             if (issue.assignee) {
+                issue.labels.push("ERR: Assignee");
```

Comment:
Yes, you can. Just that they will not have any color. You can have a look at how it looks here: https://github.com/mskd12/glassfish/issues/2879
I was doing this earlier for the tags and labels fields also.